package.json

```diff
@@ -37,6 +37,8 @@
     "babel-eslint": "4.1.6",
     "babel-loader": "6.2.0",
     "babel-plugin-react-transform": "1.1.1",
+    "babel-plugin-transform-es2015-modules-commonjs": "6.3.16",
```

Comment:
Ok, then you should remove the plugin from .babelrc as it is throwing an error when you run `npm start`. Check this issue: https://github.com/cloverfield-tools/universal-react-boilerplate/issues/61